test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalDeleteTests.cs

```diff
@@ -126,18 +131,44 @@ public async Task Given50MatchingResources_WhenDeletingConditionallyWithMultiple
 
         private async Task CreateWithIdentifier(string identifier)
         {
-            Encounter observation = Samples.GetJsonSample("Encounter-For-Patient-f001").ToPoco<Encounter>();
-
-            observation.Identifier.Add(new Identifier("http://e2etests", identifier));
-            using FhirResponse<Encounter> response = await _client.CreateAsync(observation);
-
-            Assert.Equal(HttpStatusCode.Created, response.StatusCode);
+            await _createSemaphore.WaitAsync();
+
+            try
+            {
+                Encounter observation = Samples.GetJsonSample("Encounter-For-Patient-f001").ToPoco<Encounter>();
+
+                observation.Identifier.Add(new Identifier("http://e2etests", identifier));
+
+                await Policy
+                    .Handle<FhirException>(e =>
+                        e.StatusCode == HttpStatusCode.TooManyRequests ||
+                        e.StatusCode == HttpStatusCode.ServiceUnavailable ||
+                        e.StatusCode == HttpStatusCode.BadRequest)
```

Comment:
>e.StatusCode == HttpStatusCode.BadRequest [](http://example.com/codeflow?start=24&length=41)

BadRequest doesn't feels like a right thing to retry

Reply:
This is currently what is thrown for the data rate:
![image](https://user-images.githubusercontent.com/197221/128096765-818ab829-801d-4e6f-a49b-afdd85755e62.png)